### PR TITLE
Vue/outage card

### PR DESCRIPTION
**Pull request by j0pp**

### CODE REVIEW

File: myuw/static/vue/components/cards/outage.vue [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+          if (this.non404Error(this.courseStatusCode) ||
+              this.non404Error(this.noticeStatusCode)
+              /** || non404Error(this.profileStatusCode) **/) {
+            return true;
+          }
+        }
+      }
+      /** This is the logic for instructor and employee
+      if (this.isInstructor) {
+        if (this.profileStatusCode) {
```
Comment (fanglinfang) [inline myuw/static/vue/components/cards/outage.vue:78]: This should be instructed_course data

**Review by fanglinfang — CHANGES_REQUESTED** [review]

**Comment by fanglinfang** [comment]
We also want to add unit tests with mock data.

File: myuw/static/vue/components/cards/outage.vue [hunk 2]
```diff
@@ -0,0 +1,104 @@
+<template>
+  <uw-card v-if="showOutageCard" :loaded="true">
+    <template #card-heading>
+      <h3>
+        Limited data due to technical difficulties
```
[the remark this answers is not in the document]
Comment (j0pp) [reply myuw/static/vue/components/cards/outage.vue:5]: Done by Char, thanks Char.

File: myuw/static/vue/components/cards/outage.vue (hunk 1 above)
Comment (j0pp) [reply myuw/static/vue/components/cards/outage.vue]: Done

**Comment by j0pp** [comment]
Can I get someone to review?

**Review by SarthakSingh31 — APPROVED** [review]

**Review by fanglinfang — APPROVED** [review]

**Pull request merged**